client/state/ui/editor/post/actions.js

```diff
@@ -0,0 +1,65 @@
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:ui:editor:post:actions' );
+
+export function setEditingMode( mode, modeTitle, site ) {
+	debug( 'setEditingMode', mode, modeTitle, site );
+	return { type: 'TODO' };
+}
+
+export function startEditingNew( site, postOptions ) {
+	debug( 'startEditingNew', site, postOptions );
+	return { type: 'TODO' };
+}
+
+export function startEditingExisting( site, postID ) {
+	debug( 'startEditingExisting', site, postID );
+	return { type: 'TODO' };
```

Comment:
Yes, I agree.
I desparately try to avoid any decisions in this PR series.
We should definitely ditch callbacks, but I intend these PRs to provide us framework to think about this stuff.